### PR TITLE
Minor edits for abstract and intro

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Here are some superficial edits to the Abstract and Introduction. Additional sections to follow, but seems better to split them up to minimize merge conflicts.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -107,26 +106,10 @@ The adaptive immune system utilizes antigen-specific receptors that are expresse
 These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
 After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
 
-##### Cytokine Storm
```
Comment (rando2) [inline content/02.introduction.md:110]: This text moved to another section but was never deleted.

File: content/02.introduction.md [hunk 2]
```diff
@@ -107,26 +106,10 @@ The adaptive immune system utilizes antigen-specific receptors that are expresse
...
-The acute phase response to infection results in the heavy damage of the endothelium of blood vessels, which disrupts the balance between the pro- and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
-Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death in ARDS and Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus.
-<!-- Suggestion: define distinction between Cytokine Release Syndrome and Cytokine Storm Syndrome-->
+In the case of COVID-19, there is also concern about the immune system become over-active.
+One of the main immune responses contributing to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
```
Comment (RLordan) [inline content/02.introduction.md:110]:
```suggestion
One of the main immune responses contributing to the onset of acute respiratory distress syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
```

File: content/01.abstract.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 The urgency of the pandemic has led to a rapidly emerging scientific literature on SARS-CoV-2 and COVID-19.
 This manuscript represents a collaborative effort to organize and consolidate this body of literature.
 We present information about the virus in the context of what is known about related viruses, describe the pathogenesis of COVID-19, and synthesize studies emerging about the diagnosis and treatment of COVID-19 alongside literature about related illnesses.
-We summarize and synthesize this emerging literature, with an eye towards discussing elements of the disease that will be fundamental to efforts to develop interventions.
+We summarize this emerging literature with an eye towards discussing elements of the disease that will be fundamental to efforts to develop interventions.
 Our review is a collaboratively-authored, evolving document into which we seek to incorporate the ever-expanding body of information on the topic.
-This document provides a snapshot as of August, 2020.
+This document provides a snapshot as of October, 2020.
 We continue to accept new contributions and anticipate future snapshots until technologies to mitigate the pandemic are widely deployed.
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/01.abstract.md:12]: Never mind, you say it further down!

**Review by RLordan — COMMENTED** [review]
Looks great @rando2, I can approve if required.

**Review by cgreene — APPROVED** [review]
Suggested changes are helpful - approving to indicate merge when ready

File: content/02.introduction.md [hunk 4]
```diff
@@ -37,24 +37,23 @@ This review provides a biological perspective on the SARS-CoV-2 virus and effort
...
+The first genome sequence of the virus was released on January 3, 2020 and revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
+Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
 Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV and 50% to MERS-CoV [@doi:10/ggjr43].
 However, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7], with identity between SARS-CoV-2 and RATG13 as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
 This evidence therefore suggests the SARS-CoV-2 virus may be the result of zoonotic transfer of a virus from bats to humans.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/02.introduction.md:55]:
```suggestion
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
```

**Comment by rando2** [comment]
Thank you so much @RLordan! If you don't mind approving, that would be great!

**Comment by rando2** [comment]
Appveyor seems overloaded, merging based on manubot pass

**Pull request merged**